sites/main-site/src/content/blog/2025/nf-core-colab-guide.mdx

```diff
@@ -0,0 +1,202 @@
+---
+title: "Running nf-core pipelines on Google Colab"
+subtitle: "A guide to running and interacting with nf-core pipelines using Colab and VS Code"
+pubDate: 2025-08-20T12:00:00+01:00
+headerImage: "/assets/images/blog/nf-core-colab/blogpost-thumbnail.jpg"
+headerImageAlt: "Epic Handshake between nf-core and colab"
+embedHeaderImage: true
+authors:
+    - "KurayiChawatama"
+label:
+    - "community post"
+---
+
+import { Image } from "astro:assets";
+import Profile from "@components/GitHubProfilePictureExtended.svelte";
+import colabmeme from "@assets/images/blog/nf-core-colab/colab-meme.jpg";
+import disconnectmeme from "@assets/images/blog/nf-core-colab/colab-disconnected-meme.png";
+import sleepmeme from "@assets/images/blog/nf-core-colab/dont-fall-asleep-meme.png";
+
+## Running nf-core pipelines in Google Colab
+
+Running and developing nf-core pipelines can be computationally intensive, requiring resources not easily available to students, newcomers, or participants in hands-on training workshops.
+Google Colab provides a free, accessible way to leverage powerful cloud hardware for computational tasks, making it an attractive option for students, researchers, and anyone with limited local resources.
```

Comment:
```suggestion
Google Colab provides an affordable and sometimes even free, accessible way to leverage powerful cloud hardware for computational tasks, making it an attractive option for students, researchers, and anyone with limited local resources.
```

At least if I log in with my SciLifeLab account to Colab, I get the notification that any usage will be billed to my organization. If we stress the _free_ too much, people may complain to us if they are afterward billed unexpectedly.